rs/tests/message_routing/xnet/BUILD.bazel

```diff
@@ -95,6 +98,7 @@ system_test_nns(
     env = {
         "XNET_TEST_CANISTER_WASM_PATH": "$(rootpath //rs/rust_canisters/xnet_test:xnet-test-canister)",
     },
+    extra_head_nns_tags = ["manual"],  # only run this test with the mainnet NNS.
```

Comment:
What does this flag mean?

Reply:
See the [docs](https://github.com/dfinity/ic/blob/776a21f5a35a36a15df6ecf4da3c962d1f8b2dd5/rs/tests/system_tests.bzl#L275-L297).

What this change means is that the `xnet_compatibility` test will now only run with the mainnet NNS. Previously there was an additional test (suffixed with `_head_nns`) which would run the same test against the HEAD NNS.

Since we have quite hight load during the night I decided to not run the `_head_nns` variants. Do speak up if you need the additional coverage, then I'll remove this again and we need to figure out another way to reduce CI load.

Reply:
I see. So this is only about the NNS canisters, not the versions of the images that are being used?

Reply:
Indeed, it's only about the NNS canisters.

Reply:
I see. Thanks for clarifying. It might be a good idea to explicitly mention that it is about canisters to avoid confusion.

Reply:
Good point. [Done](https://github.com/dfinity/ic/pull/1898/commits/2fbd2c6fe63e78f087869eb1d68139da0a90d8ca).